apis/addons/v1alpha1/addons_types.go

```diff
@@ -1,6 +1,7 @@
 package v1alpha1
 
 import (
+	"github.com/openshift/addon-operator/internal/ocm"
```

Comment:
We can't do this because the `apis` package is intentionally kept in a separate module to avoid import conflicts for clusters service. Just have to keep the values in-sync unfortunately.

Reply:
oh that was my IDE